src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -125,15 +120,10 @@ class ValidatorStore extends BaseStore {
     });
   }
 
-  async removeValidator({ accountAddress, isContractWallet, publicKey, operators }: {
-    accountAddress: string;
-    isContractWallet: boolean;
-    publicKey: string;
-    operators: IOperator[]
-  }): Promise<boolean> {
-    const ssvStore: SsvStore = this.getStore('SSV');
+  async removeValidator({ accountAddress, isContractWallet, publicKey, operators, liquidationCollateralPeriod, minimumLiquidationCollateral, dispatch }:
```

Comment:
In this store U can do one function instead 2.
removeValidator + bulkRemoveValidator and exitValidator + bulkExitValidators
they have a minimum difference between each other and can be improved

Reply:
look at addNewValidator